compiler-rt/test/asan/TestCases/Windows/replaced_functions.c

```diff
@@ -0,0 +1,237 @@
+// RUN: %clang_cl_asan /Od %s /Fe%t
```

Comment:
Please change this to allow the test to run on a mingw-w64 build. (Note, I have not tested it.)

```suggestion
// RUN: %clang_cl_asan %Od %s %Fe%t
```